theano/gpuarray/opt.py

```diff
@@ -1095,9 +1095,11 @@ def local_gpua_advanced_incsubtensor1(op, context_name, inputs, outputs):
             set_instead_of_inc=set_instead_of_inc)
 
 
-@register_opt('fast_compile')
-@op_lifter([tensor.AdvancedIncSubtensor])
-@register_opt2([tensor.AdvancedIncSubtensor], 'fast_compile')
+# Do not register this optimization for now, as it slows down the
+# execution by a lot in most cases.
```

Comment:
I don't agree with the work "most", but don't loose time to change it. I would more tell "in important case".

Can you confirm that the timing you did is after the optimization you did in the perform?